.ci/install_crio.sh

```diff
@@ -146,10 +146,10 @@ make BUILDTAGS="$(IFS=" "; echo "${build_union[*]}")"
 sudo -E install -D -m0755 runc "/usr/local/bin/crio-runc"
 popd
 
-echo "Set manage_network_ns_lifecycle to true"
-network_ns_flag="manage_network_ns_lifecycle"
+echo "Set manage_ns_lifecycle to true"
+network_ns_flag="manage_ns_lifecycle"
```

Comment:
As we an older version of CRI-O for openshift testing (in the Fedora job), we would need to set the value of this variable above at: https://github.com/kata-containers/tests/blob/master/.ci/install_crio.sh#L73-L77

something like: 
```
                if [ "$KUBERNETES" == "yes" ]; then
			crio_version=$(get_version "externals.crio.version")
                        network_ns_flag="manage_ns_lifecycle"
		else
			crio_version=$(get_version "externals.crio.meta.openshift")
                        network_ns_flag="manage_network_ns_lifecycle"
		fi
```